android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/RegisterUserUseCase.kt

```diff
@@ -23,6 +23,7 @@ class RegisterUserUseCase @Inject constructor(private val oneginiSDK: OneginiSDK
     operator fun invoke(identityProviderId: String?, scopes: ReadableArray?, promise: Promise) {
         val identityProvider = getIdentityProvider(identityProviderId)
 
+        //Why additional identityProviderId check?
         if (identityProvider == null && identityProviderId != null) {
```

Comment:
This should probably have a comment above this line. 
It's allowed to pass null as an identityProviderId which would result in the identityProvider also being null, and then we use the default, but when we pass an id that does not exist and we can't find it, then we throw IDENTITY_PROVIDER_NOT_FOUND